src/client/datascience/kernel-launcher/localKnownPathKernelSpecFinder.ts

```diff
@@ -28,22 +25,10 @@ import { traceInfo } from '../../common/logger';
  */
 @injectable()
 export class LocalKnownPathKernelSpecFinder extends LocalKernelSpecFinderBase {
-    private _oldKernelSpecsDeleted = false;
     private _oldKernelSpecsFolder?: string;
-    private get oldKernelSpecsDeleted() {
-        return this._oldKernelSpecsDeleted || this.memento.get<boolean>('OLD_KERNEL_SPECS_DELETED__', false);
-    }
-    private set oldKernelSpecsDeleted(value: boolean) {
-        this._oldKernelSpecsDeleted = value;
-        void this.memento.update('OLD_KERNEL_SPECS_DELETED__', value);
-    }
     private get oldKernelSpecsFolder() {
```

Comment:
Could this (and associated) go as well? Looks like it's just the trace info in the constructor using it.

Reply:
I'm leaving the logging in for now, lets remove that later.